src/strategies/MangroveOrder.sol

```diff
@@ -88,6 +89,9 @@ contract MangroveOrder is Forwarder, IOrderLogic {
 
   ///@inheritdoc IOrderLogic
   function take(TakerOrder calldata tko) external payable returns (TakerOrderResult memory res) {
+    // Checking whether order is expired
+    require(tko.expiryDate == 0 || tko.expiryDate >= block.timestamp, "mgvOrder/expired");
```

Comment:
In __lastLook__ we use:
require(exp == 0 || block.timestamp <= exp, "mgvOrder/expired");

so just for readability I think we should use the same comparison.